snippets/css.code-snippets

```diff
@@ -4,6 +4,11 @@
     "prefix": "sctheme",
     "body": ["${({ theme }) => theme.$1}"]
   },
+  "Theme's spacing shortcut for styled-components": {
+    "scope": "css",
+    "prefix": "scspacing",
+    "body": ["${({ theme }) => theme.kitt.spacing * $1}px"]
```

Comment:
```suggestion
    "body": ["${({ theme }) => `${theme.kitt.spacing * $1}px`}"]
```
https://ornikar.atlassian.net/wiki/spaces/TECH/pages/2931687636/Styled+Components+Guidelines#Put-variable-css-values-inside-the-props-handling-function